cloud/cloud.go

```diff
@@ -88,15 +88,40 @@ func (c *Client) MemberOrganizations(ctx context.Context) (orgs MemberOrganizati
 	return Get[MemberOrganizations](ctx, c, MembershipsPath)
 }
 
-// Stacks returns all stacks for the given organization.
-func (c *Client) Stacks(ctx context.Context, orgUUID string, status stack.FilterStatus) (StacksResponse, error) {
+// StacksByStatus returns all stacks for the given organization.
+func (c *Client) StacksByStatus(ctx context.Context, orgUUID string, status stack.FilterStatus) (StacksResponse, error) {
 	path := path.Join(StacksPath, orgUUID)
 	if status == stack.UnhealthyFilter {
 		path += "?status=" + status.String()
 	}
 	return Get[StacksResponse](ctx, c, path)
 }
 
+func (c *Client) GetStack(ctx context.Context, orgUUID, repo, metaID string) (StackResponse, bool, error) {
+	path := path.Join(StacksPath, orgUUID)
+	path += fmt.Sprintf("?repository=%s&metaID=%s", repo, metaID)
```

Comment:
the API uses snake case for query parameters :)
```suggestion
	path += fmt.Sprintf("?repository=%s&meta_id=%s", repo, metaID)
```

Reply:
yep, because of this issue (metaID was ignored and then only filtered by repo) I encountered some odd responses and validations that (maybe) needs some relaxing in the CLI side. I sent a message in slack.

This PR is very draft, but thanks for having a look anyway :)